crates/sui-gateway/src/unit_tests/rpc_server_tests.rs

```diff
@@ -251,3 +251,155 @@ async fn test_get_transaction() -> Result<(), anyhow::Error> {
 
     Ok(())
 }
+
+#[tokio::test]
```

Comment:
should we add these tests in a separate file? many existing tests in this file will be gone once we retire gateway

Reply:
I think most of the test in this file is still applicable to fullnode RPC, maybe we can point them to fullnode instead of gateway?